content/en/docs/faq/installation/uninstall-pluggable-components.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Uninstall Pluggable Components from KubeSphere"
+keywords: "Installer, uninstall, KubeSphere, Kubernetes"
+description: "Learn how to uninstall each pluggable component in KubeSphere."
+linkTitle: "Uninstall Pluggable Components from KubeSphere"
+Weight: 16500
+---
+
+After you [enable the pluggable components of KubeSphere](../../../pluggable-components/), you can also uninstall them using the following commands. Please back up any necessary data before you uninstall these components.
+
+## Uninstall KubeSphere App Store
+
+```bash
+kubectl delete ns openpitrix-system
+```
+
+## Uninstall KubeSphere DevOps
+
+```bash
```

Comment:
I think we should disable the component first in the cc. So does the other component.

Reply:
OK.

Reply:
[The uninstallation guide for KS v3.0.0](https://v3-0.docs.kubesphere.io/docs/faq/installation/uninstall-pluggable-components/) is missing this part of editing cc. I guess this is also applicable to it?